CONTRIBUTING.adoc

```diff
@@ -0,0 +1,93 @@
+= Contributing to Plugin Health Scoring
+
+This page provides information about contributing to Plugin Health Scoring code base.
+
+== Getting started
+
+The project is using Maven with Java 17 and Yarn to be built.
+The entire build process can be handled by Maven.
+Yarn build will bundle the front-end files only.
+
+. Clone the repository
+. Install tools
+.. Java 17.
+.. Maven 3.8.6.
+Optionally, you can use the wrapper in the repository.
+In that case, replace the `mvn` command by `./mvnw` in the command listed bellow.
+.. Node 18.10.0 and Yarn 3.2.4
```

Comment:
Is line 17 the other tools that need to be installed? 

Would it change the message if lines 15 & 16 were placed below 17 instead? Or is Maven considered the default wrapper in this case?

Reply:
Technically, you cannot do anything without Maven but Maven is not technically building everything either.. complex situation. 
The `wrapper` is only available to act as Maven and so contributors don't necessarily have to install it. The wrapper install it for them, in a sense. 

From a document syntax point of view, this list is rendered as

<img width="555" alt="Screenshot 2022-10-14 at 15 39 40" src="https://user-images.githubusercontent.com/985955/195861100-cdd859b8-c5a6-45ad-a46b-c56b176728a6.png">

So lines `15-16` are really in the list item started on line `14`.

Reply:
Okay got it, that makes more sense. In that case everything here looks good to me!